apps/experiments/utils.py

```diff
@@ -30,3 +34,15 @@ def calculate_version_number(parent_experiment_id):
         or 0
     )
     return last_version_number + 1
+
+
+def copy_event_actions(experiment_id):
+    event_actions = EventAction.objects.filter(experiment_id=experiment_id)
+    for event_action in event_actions:
+        event_action.pk = None
+        if "pipeline_id" in event_action.params:
+            old_pipeline_id = event_action.params["pipeline_id"]
+            old_pipeline = Pipeline.objects.get(id=old_pipeline_id)
+            new_pipeline = Pipeline.objects.create(name=f"{old_pipeline.name} (copy)", data=old_pipeline.data)
```

Comment:
We'll need to copy and update the `Node` model instances too.

We'll definitely want to 'tag' these as copies and prevent them from showing in the UI by default (and make them readonly). In the long run we should apply a similar version scheme to these pipelines and then we can apply a similar mechanism as routes where the user can choose what to do (create a new version or use an existing one).